ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/Analyzer.java

```diff
@@ -668,8 +671,11 @@ private void throwOnUdafs(final Expression expression) {
         public Void visitFunctionCall(final FunctionCall functionCall, final Void context) {
           final FunctionName functionName = functionCall.getName();
           if (metaStore.isAggregate(functionName)) {
-            throw new KsqlException("Use of aggregate function "
-                + functionName.text() + " requires a GROUP BY clause.");
+            analysis.addAggregateFunction(functionCall);
+            // Since this is a dummy group by, we don't actually need a correct node location
+            final NodeLocation nodeLocation = new NodeLocation(0,0);
+            analysis.setGroupBy(new GroupBy(Optional.of(nodeLocation),
+                ImmutableList.of(new IntegerLiteral(1))));
```

Comment:
thoughts on using just a single byte to save some space?

Reply:
switched to BytesLiteral